projects/packages/jetpack-mu-wpcom/src/features/wpcom-themes/wpcom-themes.php

```diff
@@ -7,70 +7,6 @@
  * @package automattic/jetpack-mu-wpcom
  */
 
-use Automattic\Jetpack\Jetpack_Mu_Wpcom;
-
-/**
- * Displays a banner before the theme browser that links to the WP.com Theme Showcase.
- */
-function wpcom_themes_show_banner() {
```

Comment:
We can't remove this function until it's removed from being called on WPCOM: fbhepr%2Skers%2Sjcpbz%2Sjc%2Qpbagrag%2Snqzva%2Qcyhtvaf%2Sjcpbz%2Qgurzrf.cuc%3Se%3Q04155113%23176%2Q179-og